docs(graalvm): graalvm examples (#2499)
kestra-io/docs: content/docs/15.how-to-guides/ruby.md

Once this has executed, both the metrics can be viewed under **Metrics**.

![metrics](/docs/how-to-guides/ruby/metrics.png)

> ## Execute GraalVM Task

Reviewer: This example is using Python in the Ruby how-to guide.

Kestra also supports GraalVM integration, allowing you to execute Ruby code directly on the JVM, with the potential for performance improvements. There are currently two tasks:
- [Eval](/plugins/plugin-graalvm/ruby/io.kestra.plugin.graalvm.ruby.eval)
- [FileTransform](/plugins/plugin-graalvm/ruby/io.kestra.plugin.graalvm.ruby.filetransform)

> In this example, the `Eval` is used to manipulate data from a previous task. GraalVM makes it easy for us to generate outputs from variables in Python with the use of the `outputs` property. Useful if you want to manipulate data and pass the new format to another task.

Reviewer: ```suggestion
In this example, the `Eval` is used to manipulate data from a previous task. GraalVM makes it easy to generate outputs from variables in Python using the `outputs` property. This is useful if you want to manipulate data and pass the new format to another task.
```

```yaml
id: parse_json_data
namespace: company.team

tasks:
  - id: download
    type: io.kestra.plugin.core.http.Download
    uri: http://xkcd.com/info.0.json

  - id: graal
    type: io.kestra.plugin.graalvm.python.Eval
    outputs:
      - data
    script: |
      data = {{ read(outputs.download.uri) }}

>       data["next_month"] = '{{ read(outputs.download.uri) | jq(".month") | first }}'.to_i + 1

Author: If anyone can get something else that works and is simple, do feel free to replace
